cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -961,3 +969,92 @@ func (ht *hcsTask) ProcessorInfo(ctx context.Context) (*processorInfo, error) {
 		count: ht.host.ProcessorCount(),
 	}, nil
 }
+
+func (ht *hcsTask) requestAddContainerMount(ctx context.Context, resourcePath string, settings interface{}) error {
+	modification := &hcsschema.ModifySettingRequest{
+		ResourcePath: resourcePath,
+		RequestType:  guestrequest.RequestTypeAdd,
+		Settings:     settings,
+	}
+	return ht.c.Modify(ctx, modification)
+}
+
+func isMountTypeSupported(hostPath, mountType string) bool {
+	// currently we only support mounting of host volumes/directories
+	switch mountType {
+	case hcsoci.MountTypeBind:
+	case hcsoci.MountTypePhysicalDisk:
+	case hcsoci.MountTypeVirtualDisk:
+	case hcsoci.MountTypeExtensibleVirtualDisk:
```

Comment:
golang switch cases don't fallthrough, was that your intention with these cases?

Reply:
not sure if I misunderstood one of the review comments - but intent was to be explicit about what mount types are not supported

Reply:
updated switch case